component/vdr/orb/vdr.go

```diff
@@ -86,6 +88,18 @@ const (
 	Recover
 )
 
+// VerifyResolutionResultType verify resolution result type.
+type VerifyResolutionResultType int
+
+const (
+	// All will verify unpublished and published operations.
+	All VerifyResolutionResultType = iota
+	// Unpublished will verify unpublished operations.
```

Comment:
It will verify document only if it has unpublished operations - comment is misleading

Reply:
done